src/common.js

```diff
@@ -7,10 +7,18 @@ export { envGlobal as global };
 
 export let baseUrl;
 
+export function setBaseUrl(url) {
+  // Verify it's a valid URL
+  const URL = require('url').URL;
+  baseUrl = new URL(url).href;
+}
+
 if (hasDocument) {
   const baseEl = document.querySelector('base[href]');
   if (baseEl)
     baseUrl = baseEl.href;
+} else if (typeof process !== 'undefined' && process.cwd) {
+  baseUrl = require('url').pathToFileURL(process.cwd() + require('path').sep).href;
```

Comment:
not sure if this is the best place to put this - also considered moving it to node-fetch.js to avoid need for feature checking.